docker/Dockerfile.cc-base-ubuntu

```
@@ -237,6 +237,10 @@ USER root
 COPY condor/condor_config /etc/condor/
 COPY condor/config.d  /etc/condor/config.d/
 
+RUN curl -L https://github.com/opensciencegrid/osg-vo-config/archive/refs/heads/master.tar.gz | \
+    tar -xz --strip-components=1 --directory=/etc/grid-security */vomses */vomsdir && \
```

When I tested on Fedora, no issues. But it seems Ubuntu 20.04 expects the `--wildcards` argument.

```suggestion
    tar -xz --strip-components=1 --directory=/etc/grid-security --wildcards */vomses */vomsdir && \
```